### DISCUSSION TITLE
How to to pull the env (Vercel declared Environment Variables) on Deployment Build?

### DISCUSSION THREAD

**Discussion by mytabworks** [vercel/vercel#10817] [Help] [answered]
### Question

I am using drizzle ORM and on the package.json script "build" I am running "next build && npx drizzle-kit push:mysql" but the command for "npx drizzle-kit push:mysql" is looking for the .env file to get the credentials for my DB, however all my declared Environment Variables are all stored on Vercel? how could I pull/generate the .env before it runs the the "build" script?

**Comment by DigitalPromiseVictor** [comment, marked as answer]
I would look into adding a script in package.json:
`prebuild: vercel env pull`

https://docs.npmjs.com/cli/v10/using-npm/scripts
https://vercel.com/docs/projects/environment-variables

**Reply by amyegan** [reply]
https://vercel.com/docs/cli/env
